openedx/features/enterprise_support/api.py

```diff
@@ -792,6 +812,23 @@ def get_enterprise_learner_data_from_db(user):
         return serializer.data
 
 
+@enterprise_is_enabled()
+def get_active_enterprise_customer_user(user):
+    """
+    Query the database to return active enterprise customer user and serialize the result. There can only be one active
+    EnterpriseCustomerUser instance against a user_id.
+    """
+    if user.is_authenticated:
+        try:
+            enterprise_customer_user = EnterpriseCustomerUser.objects.get(user_id=user.id, active=True)
+        except EnterpriseCustomerUser.DoesNotExist:
+            LOGGER.exception(
```

Comment:
We should mark it as an exception, As its normal behavior users are not linked with an enterprise.
```
LOGGER.info
```